packages/mobile/ios/celo/main.m

```diff
@@ -4,6 +4,10 @@
 
 int main(int argc, char * argv[]) {
   @autoreleasepool {
+
+    // Ignore SIGPIPE signals
+    signal(SIGPIPE, SIG_IGN);
```

Comment:
no import needed here?

Reply:
Yes that surprised me as well but Xcode did not complain. I think one of the core ios headers imports it already.